src/alr/alr-commands-test.adb

```diff
@@ -102,56 +106,158 @@ package body Alr.Commands.Test is
          Output : Utils.String_Vector;
          Start  : Time;
 
-         -------------------
-         -- Build_Release --
-         -------------------
+         -----------------
+         -- Test_Action --
+         -----------------
 
-         procedure Build_Release is
+         procedure Test_Action is
             use Ada.Directories;
             use Alire.OS_Lib.Subprocess;
             use Alire.Utils;
 
-            Alr_Args : constant String_Vector :=
-                         Empty_Vector &
-                         "get" &
-                         "--build" &
-                         "-d" &
-                         "-n" &
-                         R.Milestone.Image;
+            Docker_Prefix : constant String_Vector :=
+                              Empty_Vector
+                              & "sudo"
+                              & "docker"
```

Comment:
Why do we need docker here?

Reply:
It is a way of testing locally in the same environment used by the Linux CI github workflow, instead of in whatever local environment one has. Convenience, mostly. If only Windows/macOS could be run cross-host...